### DISCUSSION TITLE
[archived] "FAQ" docs discussion

### DISCUSSION THREAD

**Discussion by giscus** [argoproj/argo-workflows#13070] [General]
# en/release-3.4/faq/

https://argo-workflows.readthedocs.io/en/release-3.4/faq/

**Comment by jiantang99** [comment]
I am still receiving "token not valid for running mode" even after following the steps here and in https://argo-workflows.readthedocs.io/en/latest/access-token/.
Is there any other reason for this happening? I tried giving the user admin permissions, and I still get this error.

**Reply by agilgur5** [reply]
Do you have `--auth-mode=client` set for your Server?
You didn't provide any configuration so it's pretty hard to say anything meaningful
